src/templates/editor/review_revisions.html

```diff
@@ -1,91 +1,110 @@
-<h2>Review Rounds <form method="post" action="{% url 'editor_review' submission.id %}">{% csrf_token %}<button type="submit" name="new_round" class="btn btn-sm btn-accept pull-right" style="margin-bottom:5px"><i class="fa fa-plus">&nbsp;</i>Start New Round</button></h2>
+<h2>Review Rounds
+    <form method="post"
```

Comment:
Is this form closed anywhere?

Reply:
Well spotted! No it wasn't closed. I think browsers have just been guessing as to where to close it.

Reply:
It swallows up any element that follows it till it reaches the end of the document, where it will add any missing closing tags, including body and html. So be carefule if using iframe as you may end up with half your page disappearing.